Emby.Dlna/Api/DlnaServerService.cs

```diff
@@ -380,4 +286,98 @@ private object GetSubscriptionResponse(EventSubscriptionResponse response)
+    public class ProcessConnectionManagerControlRequest : IRequiresRequestStream
+    {
+        [ApiMember(Name = "UuId", Description = "Server UuId", IsRequired = false, DataType = "string", ParameterType = "path", Verb = "GET")]
+        public string UuId { get; set; }
+
+        public Stream RequestStream { get; set; }
+    }
+
+    [Route("/Dlna/{UuId}/mediareceiverregistrar/control", "POST", Summary = "Processes a control request")]
+    public class ProcessMediaReceiverRegistrarControlRequest : IRequiresRequestStream
+    {
+        [ApiMember(Name = "UuId", Description = "Server UuId", IsRequired = false, DataType = "string", ParameterType = "path", Verb = "GET")]
+        public string UuId { get; set; }
+
+        public Stream RequestStream { get; set; }
+    }
+
+    [Route("/Dlna/{UuId}/mediareceiverregistrar/events", "SUBSCRIBE", Summary = "Processes an event subscription request")]
+    [Route("/Dlna/{UuId}/mediareceiverregistrar/events", "UNSUBSCRIBE", Summary = "Processes an event subscription request")]
+    public class ProcessMediaReceiverRegistrarEventRequest
+    {
+        [ApiMember(Name = "UuId", Description = "Server UuId", IsRequired = false, DataType = "string", ParameterType = "path", Verb = "SUBSCRIBE,UNSUBSCRIBE")]
+        public string UuId { get; set; }
+    }
+
+    [Route("/Dlna/{UuId}/contentdirectory/events", "SUBSCRIBE", Summary = "Processes an event subscription request")]
+    [Route("/Dlna/{UuId}/contentdirectory/events", "UNSUBSCRIBE", Summary = "Processes an event subscription request")]
+    public class ProcessContentDirectoryEventRequest
+    {
+        [ApiMember(Name = "UuId", Description = "Server UuId", IsRequired = false, DataType = "string", ParameterType = "path", Verb = "SUBSCRIBE,UNSUBSCRIBE")]
+        public string UuId { get; set; }
+    }
+
+    [Route("/Dlna/{UuId}/connectionmanager/events", "SUBSCRIBE", Summary = "Processes an event subscription request")]
+    [Route("/Dlna/{UuId}/connectionmanager/events", "UNSUBSCRIBE", Summary = "Processes an event subscription request")]
+    public class ProcessConnectionManagerEventRequest
+    {
+        [ApiMember(Name = "UuId", Description = "Server UuId", IsRequired = false, DataType = "string", ParameterType = "path", Verb = "SUBSCRIBE,UNSUBSCRIBE")]
+        public string UuId { get; set; }
+    }
+
+    [Route("/Dlna/{UuId}/icons/{Filename}", "GET", Summary = "Gets a server icon")]
+    [Route("/Dlna/icons/{Filename}", "GET", Summary = "Gets a server icon")]
+    public class GetIcon
+    {
+        [ApiMember(Name = "UuId", Description = "Server UuId", IsRequired = false, DataType = "string", ParameterType = "query", Verb = "GET")]
+        public string UuId { get; set; }
+
+        [ApiMember(Name = "Filename", Description = "The icon filename", IsRequired = true, DataType = "string", ParameterType = "path", Verb = "GET")]
+        public string Filename { get; set; }
+    }
```

Comment:
I'm not sure if this change is necessary, I would be okay just ignoring the warning for these service class files. All these classes are all going to be removed in the near future anyways once they are migrated to the new ASP framework.

And if fact, you've mixed approaches here because the warning is disabled in the `SwaggerService` instead of reordering the classes: https://github.com/jellyfin/jellyfin/pull/2937/files#diff-726b3f928b361a5413f9386542b475fcR1

Reply:
I asked on the Riot and they told me I should minimise changes to the Emby.Server.Implementations namespace. That is why this one is renamed, but others have their warnings disabled.

However. @Bond-009 then approved a lot of file name changes anyway. That must've been after I've made this change.

Reply:
I *mostly* meant the API services in Emby.Server.Implementations
`Emby.Dlna` is one of the few projects with more API services (like this one) a pragma is fine.

Reply:
Roger. I'll revert the changes and add a pragma.

Reply:
> I asked on the Riot and they told me I should minimise changes to the Emby.Server.Implementations namespace

Maybe @Bond-009 can confirm this, but I think you can still make changes in this namespace as long as classes are not renamed. So renaming files and splitting classes up into separate files should be fine, as long as each class keeps the same name and namespace. But maybe I'm missing the reason that advice was given.

Though as mentioned above for the API services like this file, it's best to just add a `#pragma`.

Reply:
So I revert the changes I've done to these files and I can ignore your comments above about fixing instead of disabling?

Reply:
@Bond-009 and @mark-monteiro can I get a conclussion on this? I would like a definitive statement before I do these changes and I'd love to get this finished.

Reply:
Sorry, I don't think I got notifications for the last few comments for some reason.

> Roger. I'll revert the changes and add a pragma.

Yes that's what I would do for this class and the other API classes.

Reply:
Most of the api classes I touched already had pragma disables because that's what I agreed on with Bond. It was just those two in the other namespace. Which are now fixed.

Reply:
> Roger. I'll revert the changes and add a pragma.

Were you still planning on doing this?

Reply:
some changes were lost undoing the oops merge. I'll redo this one but I lost track of exactly what work got lost.

Reply:
The DLNA files should be fine now.

Reply:
Yup, looks good 👍